installer/lib/phx_new/single.ex

```diff
@@ -155,4 +194,10 @@ defmodule Phx.New.Single do
   def gen_bare(%Project{} = project) do
     copy_from project, __MODULE__, :bare
   end
+
+  def assert_live_switches!(project) do
+    unless Project.html?(project) and Project.webpack?(project) do
+      raise "cannot generate --live project with --no-html or --no-webpack. LiveView requires HTML and webpack"
```

Comment:
It may be worth supporting a version where there is no webpack dependency either. Imagine if we could skip the need for node altogether! But this can be for a later step!

Reply:
Baby steps. For now webpack is definitely required as otherwise phoenix.js and phoenix_live_view.js would have to be included on the layout, and referenced off the global Phoenix.